.travis/build.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -e
+
+# The first segment of the version number is '1' for releases < 9; then '9', '10', '11', ...
+JAVA_MAJOR_VERSION=$(java -version 2>&1 | sed -E -n 's/.* version "([0-9]*).*$/\1/p')
+if [ ${JAVA_MAJOR_VERSION} -gt 1 ] ; then
+  export JAVA_VERSION=${JAVA_MAJOR_VERSION}
+fi
+
+if [ ${JAVA_MAJOR_VERSION} -eq 1 ] ; then
+  # some parts of the workflow should be done only one on the main build which is currently Java 8
+  export MAIN_BUILD="TRUE"
+fi
+
+export PULL_REQUEST=${PULL_REQUEST:-true}
+export BRANCH=${BRANCH:-master}
+export TAG=${TAG:-latest}
+
+if [ ${JAVA_MAJOR_VERSION} -eq 1 ] ; then
+  mvn -e -V -B install
+else
+  mvn -e -V -B -Dmaven.javadoc.skip=true install
+fi
+
+
+mvn spotbugs:check
+
+# Push only releases
+if [ "$PULL_REQUEST" != "false" ] ; then
+    echo "Building Pull Request - nothing to push"
+elif [ "$TAG" = "latest" ] && [ "$BRANCH" != "master" ]; then
+    echo "Not in master branch and not in release tag - nothing to push"
+else
+    if [ "${MAIN_BUILD}" = "TRUE" ] ; then
+        echo "Pushing JARs"
+        ./.travis/push-to-nexus.sh
```

Comment:
Is the entire path needed here?

Reply:
I'm quite sure I added it for a reason. But it was long time ago (here just copied from our main repo). Do you want me to try it?

Reply:
No need to change then ;)